app/static/js/components/NetworkGraph/NetworkGraph.jsx

```diff
@@ -0,0 +1,101 @@
+import { Graph } from "react-d3-graph";
+import ReactLoading from "react-loading";
+import React from "react";
+import LoadingSpinner from "../core/LoadingSpinner/LoadingSpinner.jsx";
+
+class NetworkGraph extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null
+    };
+    this.config = {
+      height: 800,
+      panAndZoom: false,
+      staticGraph: false,
+      width: 800,
+      node: {
+        color: "#d3d3d3",
+        fontSize: 8,
+        fontWeight: "normal",
+        highlightColor: "SAME",
+        highlightFontSize: 8,
+        highlightFontWeight: "normal",
+        highlightStrokeColor: "SAME",
+        highlightStrokeWidth: 1.5,
+        labelProperty: "name",
+        mouseCursor: "pointer",
+        opacity: 1,
+        renderLabel: true,
+        size: 200,
+        strokeColor: "none",
+        strokeWidth: 1.5,
+        svg: "",
+        symbolType: "circle"
+      },
+      link: {
+        opacity: 1,
+        semanticStrokeWidth: false,
+        strokeWidth: 1.5,
+        highlightColor: "#d3d3d3"
+      }
+    };
+    this.onClickNode = this.onClickNode.bind(this);
```

Comment:
any reason to bind all of these if we won't use them yet? maybe just comment out for now